New YesWiki version that supports geojson

mrflos opened the pull request:
## Problem

- *New YesWiki version that supports geojson*

## Solution

- *just release like the upstream!*

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

nils-van-zuijlen commented on conf/app.src:
```diff
@@ -1,6 +1,6 @@
-SOURCE_SUM=1d22ab67b1d19433b85c81b58483fbb766edea04bfe5e588d489d2a1d0043404
+SOURCE_URL=https://repository.yeswiki.net/doryphore/yeswiki-doryphore-2021-05-24-7.zip
+SOURCE_SUM=1172748819c73065ed940b3e3eb531dc
 SOURCE_SUM_PRG=sha256sum
```
```suggestion
SOURCE_SUM_PRG=md5sum
```

nils-van-zuijlen commented:
!testme

mrflos commented:
!testme

ericgaspar commented:
!testme

ericgaspar commented on conf/app.src:
```diff
@@ -1,6 +1,6 @@
-SOURCE_SUM_PRG=sha256sum
+SOURCE_URL=https://repository.yeswiki.net/doryphore/yeswiki-doryphore-2021-07-16-1.zip
+SOURCE_SUM=96b79805f130a0759fbd1560a15904ec
+SOURCE_SUM_PRG=md5sum
```
We should use sha256sum for the shasum.

mrflos replied:
Ok, i will ping you when the change is effective. For now the bot will still commit the changes, with md5sum, but i will implement sha256sum soon i hope

ericgaspar replied:
How does the bot works? Is it depending on a md5 generated  upstream?

mrflos replied:
maybe bot was not the appropriate term for it, it's a repository builder, that generates zip files and md5sum (for now, because YesWiki's update manager still relies on md5), that also automates the changes for the yunohost package and commits to the yunohost repo.
the source code is there : https://github.com/YesWiki/yeswiki-build-repo

mrflos replied:
I will implemente the feature to also generate sha256sum with this builder, just i'm not sure when ;)

oncletom replied:
```suggestion
SOURCE_SUM_PRG=sha256sum
```

oncletom commented on conf/app.src:
```diff
@@ -1,6 +1,6 @@
-SOURCE_SUM=1d22ab67b1d19433b85c81b58483fbb766edea04bfe5e588d489d2a1d0043404
-SOURCE_SUM_PRG=sha256sum
+SOURCE_URL=https://repository.yeswiki.net/doryphore/yeswiki-doryphore-2021-07-27-7.zip
+SOURCE_SUM=c359ca19a88470d773483c808332e866
```
```suggestion
SOURCE_SUM=1aeb3f605d792ceee983178c7dfda97da77dbf9d9f21b60ac0febc3e7a9b55a1
```

mrflos commented:
Hi @oncletom thanks for the command line, i added it inside the building process, it's now automatically added in the source of the yunohost package.
@ericgaspar this version is now ready to be tested in order to be published (the last yeswiki yunohost version has some bugs that are now corrected ) cf. https://yeswiki.net/forum/?qa=3870/pb-cr%C3%A9ation-dun-utilisateur-automatique-dans-formulaire

ericgaspar commented:
!testme

oncletom commented:
Are any red flags remaining?

It sounds like there are no blockers left, and releasing this version would increase the quality of the distributed YesWiki version in the YunoHost app store.

ericgaspar commented:
Yolomerging
